meteor/server/api/userActions.ts

```diff
@@ -1143,6 +1150,9 @@ class ServerUserActionAPI extends MethodContextAPI implements NewUserActionAPI {
 	async packageManagerAbortExpectation(_userEvent: string, deviceId: PeripheralDeviceId, workId: string) {
 		return makePromise(() => packageManagerAbortExpectation(this, deviceId, workId))
 	}
+	async packageManagerRestartPackageContainer(_userEvent: string, deviceId: PeripheralDeviceId, containerId: string) {
+		return makePromise(() => packageManagerRestartPackageContainer(this, deviceId, containerId))
```

Comment:
For new code it would be nice for it to be primarily promise based and avoid conversions between promises and fibers when possible.
So instead of `makePromise()` packageManagerRestartPackageContainer should be async and return a Promise.

What you have done is consistent with the other packageManager methods here, so I am happy for that to be done later